production_configs/srcdep_zen20/lstchain_config.json

```diff
@@ -0,0 +1,267 @@
+    "dist"
+  ],
+
+  "disp_classification_features": [
+    "log_intensity",
+    "width",
+    "length",
+    "wl",
+    "skewness_from_source",
+    "kurtosis",
+    "time_gradient_from_source",
+    "leakage_intensity_width_2",
+    "dist"
+  ],
+
+  "particle_classification_features": [
+    "log_intensity",
+    "width",
+    "length",
+    "wl",
+    "skewness_from_source",
+    "kurtosis",
+    "time_gradient_from_source",
+    "leakage_intensity_width_2",
+    "log_reco_energy",
+    "dist"
+  ],
+
+    
+  "allowed_tels": [1],
+  "max_events": null,
+  "write_pe_image": false,
+  "mc_image_scaling_factor": 1,
+  "CameraCalibrator": {
+    "apply_waveform_time_shift": false
+  },
+  "time_sampling_correction_path": "default",
+  "image_extractor": "LocalPeakWindowSum",
+  "image_extractor_for_muons": "GlobalPeakWindowSum",
+  "LocalPeakWindowSum":{
+    "window_shift": 4,
+    "window_width":8,
+    "apply_integration_correction": true
+  },
+  "GlobalPeakWindowSum":{
+    "window_shift": 4,
+    "window_width":8,
+    "apply_integration_correction": false
+  },
+  "timestamps_pointing":"ucts",
+
+  "train_gamma_src_r_deg": [0, Infinity],
+
+  "source_dependent": true,
+  "mc_nominal_source_x_deg": 0.4,
+  "mc_nominal_source_y_deg": 0.0,
+
+  "volume_reducer":{
+    "algorithm": null,
+    "parameters": {
+    }
+  },
+  "calibration_product": "LSTCalibrationCalculator",
+
+  "LSTCalibrationCalculator":{
+    "systematic_correction_path": null,
+    "squared_excess_noise_factor": 1.222,
+    "flatfield_product": "FlasherFlatFieldCalculator",
+    "pedestal_product": "PedestalIntegrator",
+    "PedestalIntegrator":{
+      "sample_size": 10000,
+      "sample_duration":100000,
+      "tel_id":1,
+      "time_sampling_correction_path": null,
+      "charge_median_cut_outliers": [-10,10],
+      "charge_std_cut_outliers": [-10,10],
+      "charge_product":"FixedWindowSum",
+      "FixedWindowSum":{
+        "window_shift": 6,
+        "window_width":12,
+        "peak_index": 18,
+        "apply_integration_correction": false
+      }
+    },
+    "FlasherFlatFieldCalculator":{
+      "sample_size": 10000,
+      "sample_duration":100000,
+      "tel_id":1,
+      "time_sampling_correction_path": null,
+      "charge_product":"LocalPeakWindowSum",
+      "charge_median_cut_outliers": [-0.5,0.5],
+      "charge_std_cut_outliers": [-10,10],
+      "time_cut_outliers": [2,38],
+      "LocalPeakWindowSum":{
+        "window_shift": 5,
+        "window_width":12,
+        "apply_integration_correction": false
+      }
+    }
+  },
+  "waveform_nsb_tuning":{
+    "nsb_tuning": false,
+    "nsb_tuning_ratio": 0.52,
+    "spe_location": "lstchain/data/SinglePhE_ResponseInPhE_expo2Gaus.dat"
+  },
+  "image_modifier": {
+      "increase_nsb": false,
+      "extra_noise_in_dim_pixels": 0,
+      "extra_bias_in_dim_pixels": 0,
+      "transition_charge": 0,
+      "extra_noise_in_bright_pixels": 0,
+      "increase_psf": false,
```

Comment:
Are you sure you want this to be False?
Just making sure :)

Reply:
Yes, because, we should have some standard source-dependent DL1 files as a reference for the tuning, as we did earlier for zenith 40 deg, source independent MC production.

So, these 2 PRs are the primer for another set of PRs for 
1. Source-dependent, PSF tuning, zenith 20 and 40 deg
2. Source-dependent, PSF + NSB tuning, zenith 20 and 40 deg (depending on the selected NSB)

Reply:
I am not sure how much is too much for MC production while we wait for the all-sky MC set ;-)

Reply:
Ah right thats a new base, not dl1ab.
My bad

Reply:
> I am not sure how much is too much for MC production while we wait for the all-sky MC set ;-)

Guess we have some things to discuss in the call